### PR TITLE
Digital Shadows Phantom app bug fixes v2.0.1

### PR DESCRIPTION
**Pull request by shikhin-metron**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [X] Testing of all the changes has been performed (for bug fixes / features)
- [X] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [X] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [X] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [ ] Verify all checks are passing.
- [X] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [X] Do NOT submit updates to dependencies unless it fixes an issue.
#### Please check the type of change your PR introduces:
- [ ] New App
- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Security Considerations (REQUIRED)
- [ ] If you are exposing any endpoints using a [REST handler](https://docs.splunk.com/Documentation/SOAR/current/DevelopApps/RESTHandlers), 
  please document them in the `readme.html`.
- [ ] If this is a new connector or you are adding new actions
    - [ ] Please document in the `readme.html` all methods (eg, OAuth) used to authenticate 
      with the service that the connector is integrating with.
    - [ ] If any actions are unable to run on SOAR Cloud, please document this in the `readme.html`.
- [ ] Are you introducing any new cryptography modules? If yes, please elaborate their purpose:
- [ ] Are you are accessing the file system? If yes, please verify that you are only accessing paths returned through 
the [Vault](https://docs.splunk.com/Documentation/SOARonprem/5.2.1/DevelopApps/AppDevAPIRef#Vault) API.
- [ ] Are you are marking code to be ignored by Semgrep with [`nosemgrep`](https://semgrep.dev/docs/ignoring-files-folders-code/#ignoring-code-through-nosemgrep)?
If yes, please provide justification in an additional comment next to the ignored code.

  

## Release Notes (REQUIRED)
- Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.

## What is the current behavior? (OPTIONAL)
- Describe the current behavior that you are modifying.

## What is the new behavior? (OPTIONAL)
- Describe the behavior or changes that are being added by this PR.

## Other information (OPTIONAL)
- Any other information that is important to this PR such as screenshots of how the component looks before and after the change.

## Pay close attention to (OPTIONAL)
- Any specific code change or test case points which must be addressed/reviewed at the time of GA release.

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

**Comment by pdesai-crest** [comment]
@shikhin-metron I see that `Unidecode` library is causes static tests to fail because it carries a GPL license which is not approved splunk. You can use a library which is MIT licensed or license similar to that

This issue can help you maybe: https://github.com/wagtail/wagtail/issues/3311 
TLDR; they used [`anyascii`](https://pypi.org/project/anyascii/) instead of `unidecode` which uses `ISC license` (approved by splunk) instead of `GPL`

Also can you use pre-commit hooks in this repo so that wheel structure is according to standards.

**Comment by shikhin-metron** [comment]
> @shikhin-metron I see that `Unidecode` library is causes static tests to fail because it carries a GPL license which is not approved splunk. You can use a library which is MIT licensed or license similar to that
> 
> This issue can help you maybe: [wagtail/wagtail#3311](https://github.com/wagtail/wagtail/issues/3311) TLDR; they used [`anyascii`](https://pypi.org/project/anyascii/) instead of `unidecode` which uses `ISC license` (approved by splunk) instead of `GPL`
> 
> Also can you use pre-commit hooks in this repo so that wheel structure is according to standards.

Hi @pdesai-crest , thanks, I will look into it and add it instead of unidecode. Can you please tell me more about the file manipulation function failure as well? How can I resolve it?

**Comment by pdesai-crest** [comment]
@shikhin-metron, we will take care of that failure, Thanks!!

**Comment by shikhin-metron** [comment]
Hi @pdesai-crest , added the anyascii library instead of unidecode, but I still see that license issue. What might be the problem?

**Comment by dgopani-crest** [comment]
@shikhin-metron, We will take care of this failure. Thanks!

**Comment by mishalp-crest** [comment]
Hi @shikhin-metron, As we have to do dev check changes, could you please provide us (@mishalp-crest, @pdesai-crest, @dgopani-crest ) write access to your repo?

**Comment by shikhin-metron** [comment]
@mishalp-crest can you please fork it instead? The process of giving write access is not that simple on our end. If there is another alternative, I am happy to try.

**Comment by ds-jdcockrill** [comment]
Just to help @shikhin-metron out - it's me that will need to add access - feel free to tag me in future comments so I get a ping in my e-mail inbox.

I sent @shikhin-metron back to ask about forking because I'll have to buy more 'outside contributor' licenses to give more users access to the DS fork of this repository. If we can avoid that by you forking and raising a PR, I'd rather do that if possible.

Edited to clarify which repo I can give access to.

**Review by brian-phantom — APPROVED** [review]

**Review by tonyc-phantom — APPROVED** [review]

**Comment by mishalp-crest** [comment]
@shikhin-metron @ds-jdcockrill
Ok, we will manage
